app/templates/custom-elements/remote-screen.html

```diff
@@ -108,8 +118,28 @@
           screenImg.addEventListener("contextmenu", (evt) => {
             evt.preventDefault();
           });
-
-          window.addEventListener("resize", this.onWindowResize);
+          let isTouchScreen = false;
+          this.shadowRoot.addEventListener("touchend", () => {
+            isTouchScreen = true;
+          });
+          this.shadowRoot.addEventListener("click", () => {
+            if (isTouchScreen) {
+              this.shadowRoot.getElementById("mobile-keyboard-input").focus();
+            }
+          });
+          this.shadowRoot
+            .getElementById("mobile-keyboard-input")
+            .addEventListener("keyup", (evt) => {
+              if (evt.keyCode == 229) {
+                if (evt.target.innerHTML == "&nbsp;") {
+                  evt.target.innerHTML = " ";
+                }
+                sendTextInput(evt.target.innerHTML);
```

Comment:
Why `innerHTML` instead of `innerText`? 

The current implementation doesn't properly forward `<` or `>`. I think what we want is:

```javascript
if (evt.target.innerHTML === "&nbsp;") {
  sendTextInput(" ");
} else {
  sendTextInput(evt.target.innerText);
}
```

Reply:
That doesn't fix anything its essentially doing the same thing. I can change it just so its clearer. It would be the same thing as doing the same thing below.

```
sendTextInput(evt.target.innerHTML === "&nbsp? " ": evt.target.innerText);
```